# ==============================================================================
# 描述: 团队内部广播的“心跳”消息，用于队友间共享自身精确状态。
#      主要用于维持队形、避免碰撞和基本态势感知。
# ==============================================================================
std_msgs/Header header

int32 sender_id               
float32 broadcast_range
  
DynamicState dynamic_state      # 发送者完整的状态信息
bool is_alive                              # 当前的存活状态